### DISCUSSION TITLE
Power Platform CLI: pac admin list-user-settings / update-user-settings

### DISCUSSION THREAD

**Discussion by Laskewitz** [microsoft/powerplatform-build-tools#459] [Ideas]
I would like to have commands available to change user settings, like timezone, language etc on behalf of a user.

# Examples: 

- `pac admin update-user-settings --user-principal-name <UPN> --key <key> --value <value>`
- `pac admin update-user-settings --input-file ./user-settings.csv/json`

user-settings.json should contain an array of items with UPN, Key and Value columns.

# Blog about the user settings
https://www.crmcrate.com/feeds/bulk-update-user-personal-settings-from-backend-in-dynamics-365/

**Comment by anttipajunen** [comment]
Great idea!
